### PR TITLE
19 implement button fast multiple clicks

### PR DESCRIPTION
**Pull request by casainho**

### CODE REVIEW

File: src/display/KT-LCD3/lcd.c [hunk 1]
```diff
@@ -1286,56 +1299,6 @@ void lcd_execute_menu_config_submenu_various (void)
         lcd_print (ui8_temp, ODOMETER_FIELD, 1);
       }
     break;
-
-    // PAS max cadence
```
Comment (lightcore-x) [inline src/display/KT-LCD3/lcd.c:1290]: Was this accidentally removed? PAS max cadence is still used but can no longer be configured...

**Comment by lightcore-x** [comment]
The buttons are now responding a little bit slower but this isn't really annoying. I think this feature greatly increases the options with this limited amount of buttons! I'm not sure how the odometer field works exactly and how the trip meter can be cleared. I couldn't get it to work but I must be doing something wrong obviously!

File: src/display/KT-LCD3/lcd.c (hunk 1 above)
Comment (lightcore-x) [reply src/display/KT-LCD3/lcd.c:1290]: @casainho just confirmed that's it's no longer needed. We should also remove the pas_max_cadence value from EEPROM and serial protocol later.

Comment (lightcore-x) [reply src/display/KT-LCD3/lcd.c:1290]: Removed pas_max_cadence

**Comment by casainho** [comment]
I don't know if is possible to improve this added delay to click event...

I also developed the fast click + fast click but was to easy to trigger instead the click event, so I forgot that option.

**Pull request merged**